bokehjs/src/coffee/callback/customjs.coffee

```diff
@@ -1,5 +1,5 @@
 _ = require "underscore"
-Model = require "../models/model"
+Model = require "../model/model"
```

Comment:
It's all the model subclasses are already in `bokeh.models` (plural) on the python side. It's arguable whether `bokeh.model.Model` should have been `bokeh.models.Model` but I couldn't make it work, and it doesn't seem unreasonable for the overall base class to be separate.